docs/source/commands/peersnapshot.md

```diff
@@ -0,0 +1,121 @@
+# peer snapshot
+
+The `peer snapshot` command allows administrators to perform snapshot related
+operations on a peer, such as submit a snapshot request, cancel a snapshot request
+and list pending requests. Once a snapshot request is submitted for a specified
+block number, the snapshot will be automatically generated when the block number is
+committed on the channel.
+
+## Syntax
+
+The `peer snapshot` command has the following subcommands:
+
+  * cancelrequest
+  * listpending
+  * submitrequest
+
+## peer snapshot cancelrequest
```

Comment:
For some reason the peer snapshot commands are not showing up in the command reference page https://hyperledger-fabric--2004.org.readthedocs.build/en/2004/command_ref.html

Did you run  make help-docs to generate the markdown content and push all of the changed files to GitHub? If so we might need to get Brett involved to see why it is not rendering.
https://hyperledger-fabric.readthedocs.io/en/latest/docs_guide.html#adding-a-new-cli-command

Reply:
Yes, I ran `make help-docs` to generate the markdown files and pushed all the changed files to github. However, command_ref.rst is not updated. Therefore, I manually updated it and pushed to this PR.